packages/patternfly-4/react-core/src/components/Pagination/OptionsToggle.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import styles from '@patternfly/patternfly/components/OptionsMenu/options-menu.css';
+import { css, getModifier } from '@patternfly/react-styles';
+
+const OptionsToggle = ({ items, firstIndex, lastIndex, itemCount, widgetId, onToggle, isOpen }) => (
+  <div className={css(styles.optionsMenuToggle, getModifier(styles, 'plain'), getModifier(styles, 'text'))}>
+    <span className={css(styles.optionsMenuToggleText)}>
+      <b>{firstIndex} - {lastIndex}</b> of <b>{itemCount}</b> {items}
```

Comment:
Should use `strong` tag instead, also need to be able to customize I think. Using the template string builder from utility would be good